lcli/src/generate_bootnode_enr.rs

```diff
@@ -25,10 +25,21 @@ pub fn run<T: EthSpec>(matches: &ArgMatches) -> Result<(), String> {
         ));
     }
 
+    // TODO: allow using dual stack.
```

Comment:
Libp2p supports this right?

Reply:
this is the `generate_bootnode_enr` so I think this is not relevant here. I'm disallowing dual stack in this cases for now, but to answer the question yes of course